docs/pages/repo/docs/core-concepts/monorepos/configuring-workspaces.mdx

```diff
@@ -0,0 +1,181 @@
+---
+title: Configuring Workspaces
+description: Turborepo lets you configure each workspace
+---
+
+import Callout from "../../../../../components/Callout";
+
+# Configuring Workspaces
+
+Most monorepos can declare a `turbo.json` in the root directory with a uniform
+set of tasks that apply to all workspaces. But sometimes, a monorepo can contain
+workspaces that need to configure their tasks differently. To accommodate this,
+Turborepo enables you to extend the root configuration with a `turbo.json` in
+any workspace. This flexibility enables a more diverse set of apps and packages
+to co-exist, and allows workspace owners to maintain specialized tasks and
+configuration without affecting other inhabitants of the monorepo.
+
+## How it Works
+
+To override any of the configuration for a task, add a `turbo.json` file in any
+workspace of your monorepo with a top level `extends` key:
+
+```jsonc filename="turbo.json"
+{
+  "extends": ["//"],
+  "pipeline": {
+    // custom configuration for the workspace here
```

Comment:
I think we can bring slightly more color here. Something like...

// Overrides for this workspace
// dependsOn, inputs, outputs, etc.

Reply:
added some more comments, but refraining from adding speciifc keys, because (1) they get out of date, and (2) there isn't anything to compare specific keys to in this intro section